Add a brief description of IFR and CFR (#637)
greenelab/covid19-review: content/08.transmission.md

They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread. 

#### Estimating the Fatality Rate

Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.

> The mortality rate of greatest interest would be the number of fatalities over the number of people infected.

Reviewer: ```suggestion
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
```

One metric reported is often case fatality rate (CFR), which simply compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all exposed or all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.

> Metanalyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].

Reviewer 2: ```suggestion
Metaanalyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
```

> Metaanalyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].

Reviewer 3: ```suggestion
Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
```

Reviewer 3: Looks awesome

Author: Thank you so much!
